streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableRepartitionMap.java

```diff
@@ -154,27 +154,25 @@ public void process(final Record<K, Change<V>> record) {
             final KeyValue<? extends K1, ? extends V1> oldPair = record.value().oldValue == null ? null :
                 mapper.apply(record.key(), record.value().oldValue);
 
+            final boolean isLatest = record.value().isLatest;
```

Comment:
Should we move this before the `if` and use instead of `!record.value().isLatest` ?

Reply:
Updated in https://github.com/apache/kafka/pull/13615.